tests/apitest/src/CoreGraphics/CGDisplay.cs

```diff
@@ -79,21 +79,21 @@ public void CGDisplayShouldCaptureAndReleaseDisplays ()
 		{
 			int error;
 
-			error = CGDisplay.Capture (mainDisplayId);
-			Assert.Equals (0, error);
+			//error = CGDisplay.Capture (mainDisplayId);
+			//Assert.AreEqual (0, error);
```

Comment:
Better add a `[Ignore ("Affect computer while executing")]` and not comment each assert. Even better would be to define a `[Category]` that is only used on bots, not locally (by default).